Cargo.toml

```diff
@@ -85,13 +85,17 @@ members = [
     "examples/tour",
     "examples/tooltip",
     "examples/url_handler",
+    "examples/menu",
 ]
 
 [dependencies]
 iced_core = { version = "0.4", path = "core" }
 iced_futures = { version = "0.3", path = "futures" }
 thiserror = "1.0"
 
+[patch.crates-io]
+winit = { git = "https://github.com/iced-rs/winit", rev = "e351421a32bf01b428325dde44dea39ee2656153"}
```

Comment:
Use a patch so that we don't need to modify `iced-rs/glutin` while developing this branch, should be removed before merging.